### PR TITLE
Fix bidirectional data

### PR DESCRIPTION
**Pull request by mapedraza**

### CODE REVIEW

File: lib/plugins/bidirectionalData.js [hunk 1]
```diff
@@ -75,9 +75,7 @@ function extractVariables(item) {
```
Comment (fgalan) [inline lib/plugins/bidirectionalData.js:75]: CNR entry missing?

Comment (mapedraza) [reply lib/plugins/bidirectionalData.js:75]: The PR destination is another branch (the one that removes Legacy support). Final CNR entry should be included there with all the details

Comment (fgalan) [reply lib/plugins/bidirectionalData.js:75]: Ok. NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**